n3fit/src/n3fit/backends/keras_backend/callbacks.py

```diff
@@ -75,9 +76,15 @@ def __init__(self, stopping_object, log_freq=100):
         super().__init__()
         self.log_freq = log_freq
         self.stopping_object = stopping_object
+        self._current_loss = None
+
+    def on_epoch_begin(self, epoch, logs=None):
+        # TODO This is an unnecessary performance hit, just for testing
+        self._current_loss = self.model.compute_losses()
```

Comment:
I guess this should go out again? Alongs with the `logs = self._current_loss` below?